en/flight_controller/snapdragon_flight_troubleshooting.md

```diff
@@ -164,6 +85,84 @@ Then configure station mode:
 reboot
 ```
 
+## Using the cameras on the Snapdragon Flight
```

Comment:
should we have a separate section for this since it is not really troubleshooting?

Reply:
Minor point, but try use first level capitalization for headings - so this would be *Using the Cameras on the Snapdragon Flight* (we ignore the little words). This is not vital, but consistency across a document is.